components/UserOverview/UserOverview.stories.tsx

```diff
@@ -2,7 +2,7 @@ import { Meta, Story } from "@storybook/react";
 import { User } from "proto/api_pb";
 import users from "test/fixtures/users.json";
 
-import { ProfileUserProvider } from "../hooks/useProfileUser";
+import { ProfileUserProvider } from "../../features/profile/hooks/useProfileUser";
```

Comment:
We should prefer absolute imports here to ease future changes if needed rather than having to guess how many `..` levels to go up